### PR TITLE
[button] finalize schema

### PR DESCRIPTION
**Pull request by maimonav**

### CODE REVIEW

File: packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_button.proto [hunk 1]
```diff
@@ -6,21 +6,29 @@ import "wix/rich_content/v1/common.proto";
...
+    message Styles {
+        message Colors {
+            google.protobuf.StringValue text_color = 1; // hex color
+            google.protobuf.StringValue border_color = 2; // hex color
+            google.protobuf.StringValue background_color = 3; // hex color
+        }
+
+        message BorderStyles {
+            google.protobuf.Int32Value border_width = 1; // number of pixels
+            google.protobuf.Int32Value border_radius = 2; // number of pixels
```
Comment (shaulgo) [inline packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_button.proto:18]:
```suggestion
            google.protobuf.StringValue text = 1; // hex color
            google.protobuf.StringValue border = 2; // hex color
            google.protobuf.StringValue background = 3; // hex color
        }

        message Border {
            google.protobuf.Int32Value width = 1; // number of pixels
            google.protobuf.Int32Value radius = 2; // number of pixels
```

File: packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_button.proto [hunk 2]
```diff
@@ -6,21 +6,29 @@ import "wix/rich_content/v1/common.proto";
...
+        }
+        BorderStyles border_styles = 1;
```
Comment (shaulgo) [inline packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_button.proto:20]:
```suggestion
        BorderStyles border = 1;
```

**Review by shaulgo — CHANGES_REQUESTED** [review]

File: packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_button.proto (hunk 1 above)
Comment (maimonav) [reply packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_button.proto]: Done.

File: packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_button.proto (hunk 2 above)
Comment (maimonav) [reply packages/ricos-schema/web/src/main/proto/wix/rich_content/v1/plugin_button.proto]: Done.

**Comment by maimonav** [comment]
@shaulgo Anything else?

**Review by oded-soffrin — APPROVED** [review]

**Pull request merged**